src/WCSimWCDigitizer.cc

```diff
@@ -5,7 +5,7 @@
 // for memset
 
 #ifndef NPMTS_VERBOSE
-#define NPMTS_VERBOSE 10
+// #define NPMTS_VERBOSE 10
```

Comment:
Not sure why you've turned that off? It's only used in printouts, and only when `WCSIMWCDIGITIZER_VERBOSE` is on